src/foam/core/Slot.js

```diff
@@ -557,7 +557,7 @@ foam.CLASS({
       this.cleanup_ = {
         detach: function() {
           for ( var i = 0 ; i < subs.length ; i++ ) {
-            subs[i].detach();
+            if ( subs[i] ) subs[i].detach();
```

Comment:
How is this possible? Only if someone calls sub(null). Maybe better to override the sub() method and ignore null values there then remove this check. We could also log when it happens so developers would know to fix their problem.